modules/ptp-dual-nics.adoc

```diff
@@ -29,3 +29,11 @@ Dual NIC boundary clock::
 For 5G telco networks that deliver mid-band spectrum coverage, each virtual distributed unit (vDU) requires connections to 6 radio units (RUs). To make these connections, each vDU host requires 2 NICs configured as boundary clocks.
 +
 Dual NIC hardware allows you to connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding the downstream clocks.
+
+Highly available dual NIC boundary clock::
```

Comment:
This need to be worded to say it;s not BC HA, but rather SystemClock HA. High Availability of the system clock when there are multiple time sources on different NICs